src/civic/variant.js

```diff
@@ -297,10 +282,10 @@ const uploadNormalizedVariant = async (conn, normalizedVariant, feature) => {
                 [reference2] = await _entrezGene.fetchAndLoadBySymbol(conn, normalizedVariant.reference2.name);
             }
         }
-        content.reference1 = rid(reference1);
+        content.reference1 = rid(reference1, true);
 
         if (reference2) {
-            content.reference2 = rid(reference2);
+            content.reference2 = rid(reference2, true);
```

Comment:
similarly, if we have a reference2 it should also never be null

Reply:
ok